GAME/LARA.C

```diff
@@ -490,9 +503,11 @@ void lara_col_reach(struct ITEM_INFO *item, struct COLL_INFO *coll)//18D0C, 18E4
 	S_Warn("[lara_col_reach] - Unimplemented!\n");
 }
 
-void lara_as_reach(struct ITEM_INFO *item, struct COLL_INFO *coll)//18CE0, 18E14
+void lara_as_reach(struct ITEM_INFO *item, struct COLL_INFO *coll)//18CE0(<), 18E14(<) (F)
 {
-	S_Warn("[lara_as_reach] - Unimplemented!\n");
+	camera.target_angle = 15470;
+	if (item->fallspeed >= 132)
```

Comment:
For if statements, please consider using braces around the code that is executed upon condition success. This is to ensure the coding style is consistent.